ush/retrieve_data.py

```diff
@@ -300,13 +301,10 @@ def get_file_templates(cla, known_data_info, data_store, use_cla_tmpl=False):
                 if "sfc" in tmpl:
                     del file_templates[format]['fcst'][i]
 
-    if use_cla_tmpl:
-        file_templates = cla.file_templates if cla.file_templates else file_templates
-
```

Comment:
It's unclear why we don't still want this logic. If we do provide the templates via command line and want to use them, it seems this would no longer allow that option.

Reply:
reverted. removed in error.